server/aesdsocket.c

```diff
@@ -542,13 +571,17 @@ void log_timestamp()
     lseek(tmp_fd, 0, SEEK_END);
 
     // atomically write timestamp
+    #ifndef USE_AESD_CHAR_DEVICE
```

Comment:
@MaanasMDK you shouldn't need these extra #ifndef's here since you've got the entire function removed above.

Reply:
Yes, it is redundant.